pipelines/models/onnx-mnist/README.md

```diff
@@ -0,0 +1,22 @@
+# ONNX MNIST
+## Description
+This model predicts handwritten digits using a convolutional neural network (CNN).
+
+## Dataset
+The model has been trained on the popular MNIST dataset.
+
+## Source
+The model is trained in CNTK following the tutorial CNTK 103D: Convolutional Neural Network with MNIST. Note that the specific architecture used is the model with alternating convolution and max pooling layers (found under the "Solution" section at the end of the tutorial).
+## Test
+### Run OVMS with the model inside
+Execute the following command from the [pipelines](../../) folder:
+```
+docker run -d -u $(id -u):$(id -g) --rm -v ${PWD}/ovms:/model:Z -p 9000:9000 openvino/model_server:latest --model_name onnx_mnist --model_path /model/onnx-mnist --port 9000
```

Comment:
This should be mounting the `-v ${PWD}/models:/model:Z`, correct?

Reply:
yes, so if you are using a system with `selinux` we will avoid issues related to file permissions.

Reply:
@davidesalerno Just to clarify, the question was related to the host  directory name you included in the example `ovms --> models`
```suggestion
docker run -d -u $(id -u):$(id -g) --rm -v ${PWD}/models:/model:Z -p 9000:9000 openvino/model_server:latest --model_name onnx_mnist --model_path /model/onnx-mnist --port 9000
```